src/libraries/System.Drawing.Common/ref/System.Drawing.Common.csproj

```diff
@@ -1,9 +1,13 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent);netcoreapp3.0</TargetFrameworks>
     <ExcludeCurrentNetCoreAppFromPackage>true</ExcludeCurrentNetCoreAppFromPackage>
     <Nullable>enable</Nullable>
   </PropertyGroup>
+  <!-- We conditionally apply net5.0+ obsoletions -->
+  <PropertyGroup Condition="!($(TargetFramework.StartsWith('netcoreapp')) or $(TargetFramework.StartsWith('netstandard')) or $(TargetFramework.StartsWith('net4')))">
```

Comment:
This is wrong as the asset we ship as part of the package is the netcoreapp3.0 asset we don't ship the 5.0 asset. So instead of shipping a 5.0 asset just for this purpose I would suggest just making it obsolete for all tfms. @ericstj, @jeffhandley  I believe this is the pattern we've followed in other OOBs?

Reply:
Hmmm... I'll look again at the other OOBs and back at previous PRs...

Reply:
@safern is right; `DirectoryServices` is our precedent for that. https://github.com/dotnet/runtime/pull/40756/files

There, we marked the APIs as obsolete downlevel and included the `ObsoleteAttribute` class in the build as `internal`, allowing the diagnostic id and URL to flow through.  It was only for APIs were we build a net5.0+ version that we conditionally marked them as obsolete in the new version.

@JeremyKuhne, let me know if you'd like me to pull down your branch, apply this change, and push to your branch after testing the build locally.

Reply:
@safern Does this match your expectations? https://github.com/jeffhandley/runtime/commit/2cfd0e3a9941943da641ef2e974d5603815d091f

Reply:
Connected with @safern offline. He gave the 👍 .